### DISCUSSION TITLE
Eufy Bluetooth locks??

### DISCUSSION THREAD

**Discussion by jonahgcarpenter** [fuatakgun/eufy_security#420] [Ideas]
With the implementation of the new Bluetooth integration within Home Assistant it would be very cool to see locks (without the wifi bridge) become usable within the Eufy integration via bluetooth

**Comment by fuatakgun** [comment]
New implementation of Bluetooth in ha core doesn't mean anything, the solution was there since many years as a custom integration. Actually, custom BLE integration supports hundreds of sensors, while core only supports a couple.

On the other hand, i hope, for security purposes, it won't be easily deciphered and integrated into home assistant. In essence, ble integration developers decipher the data between phone and device and use the same commands from Bluetooth dongle of server. If it is easily replicable, any other external bad actor could do the same, right?
